app/session-stores/application.ts

```diff
@@ -1,3 +0,0 @@
-import SessionStorage from 'ember-simple-auth/session-stores/session-storage';
```

Comment:
Why was this deleted?

Reply:
So we can switch back to e-simple-auth's default `AdaptiveStore`, which will use `localStorage` when available. `localStorage` is shared across tabs/windows, so when the user logs in/out in one tab, the authenticator's `restore()` is called in all other tabs.

`OsfCookie.restore()` used to be kinda broken, and using `sessionStorage` helped sidestep its problems. This PR fixes it, so we can make use of that shared state again.

Reply:
No, [it's pretty unintuitive](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage):
> Opening a page in a new tab or window will cause a new session to be initiated

So `sessionStorage` sticks around within one tab/window. If you navigate away and come back within the same tab, you'll have the same `sessionStorage`, but if you open anything in a new tab, it'll be a shiny new one.

Reply:
Oh, there's [a ticket for this](https://openscience.atlassian.net/browse/EMB-268)

Reply:
That's fair. 2 birds, 1 stone for you. 👍